model.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"context"
+
+	"github.com/filecoin-project/sentinel-tick/quotetracker"
+	pg "github.com/go-pg/pg/v10"
+)
+
+const schema = "filquotes"
+
+// Quote stores FIL price information.
+type Quote struct {
+	//lint:ignore U1000 hit for go-pg
+	tableName struct{} `pg:"filquotes.fil_usd_quotes"`
+	Height    int64    `pg:",pk,use_zero,notnull"`
+	Price     float64  `pg:",use_zero,notnull"`
+	Exchange  string   `pg:",pk,use_zero,notnull`
+	Currency  string   `pg:",pk,use_zero,notnull`
+}
```

Comment:
@iand would this be ok for schema?

Reply:
I wouldn't use a float for currency. Use an int64 to represent the smallest unit you care about in the currency, e.g. milli cents